test/hashmap_shm_test.cpp

```diff
@@ -0,0 +1,118 @@
+#include <sys/types.h>
```

Comment:
hashmap_shm and hashmap test can they be mergered?

Reply:
Yeah, I'll combine the tests